src/confirmationModal.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  ReactNode,
+  FormEvent,
+  PropsWithChildren,
+  forwardRef,
+  useState,
+  useRef,
+  useImperativeHandle,
+} from "react";
+import { bug, ModalProps, ModalHandle, Modal, Spinner, Button, boxError } from ".";
+import { currentRef } from "./utilFunc";
+
+
+type ConfirmationModalProps = Omit<ModalProps, "closable" | "title"> & {
+  title?: string;
+  buttonContent: ReactNode;
+  onSubmit?: () => void;
+  text: {
+    generalActionCancel: string,
+    generalActionClose: string,
+    manageAreYouSure: string,
```

Comment:
As with the other translations: remove the `generalAction` and `manage` prefixes. I think `cancel`, `close` and `areYouSure` are sufficiently clear.